asm/istio/options/cni-managed.yaml

```diff
@@ -12,6 +12,116 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: dataplanecontrols.mesh.cloud.google.com
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.state
```

Comment:
Is this the correct order?
Revision should be first.

Reply:
I copied it from https://gke-internal-review.googlesource.com/c/istio/istio/+/297502. @kyessenov ^^

Reply:
I wasn't aware revision was first. I requested to add revision in CL.

Reply:
I created cl in g3 to reorder it as: revision, version, status and age

Reply:
Will update once the internal CL lands.